examples/polypharmacy_gripnet/configs/PoSE_MINI-GripNet.yaml

```diff
@@ -1,5 +0,0 @@
-SOLVER:
-  BASE_LR: 0.01
-  WARMUP: False
-  WARMUP_EPOCHS: 5
-  MAX_EPOCHS: 10  # For quick testing
```

Comment:
It will be good to keep this (edit as you see needed) as a fast version config and also show how to customize the config, e.g. add to ReadMe saying for a quick testing, how to use this config. 
This could be a config for the Notebook tutorial that you will work on next, which should run within minutes.
The objective is to demonstrate the flow, not to getting the best performance.